src/app/components/pages/node-url.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, OnDestroy } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
import { Component, OnInit, OnDestroy } from "@angular/core";
```